cmd/executor/cmd/root.go

```diff
@@ -206,6 +207,17 @@ func resolveDockerfilePath() error {
 	return errors.New("please provide a valid path to a Dockerfile within the build context with --dockerfile")
 }
 
+// resolveEnvironmentBuildArgs replace build args without value by the same named environment variable
+func resolveEnvironmentBuildArgs(arguments []string, resolver func(string) string) {
```

Comment:
So this will take an argument like `FOO` and replae it with `FOO=FOO` ?

Reply:
@cvgw this will take the argument array such as `[ "foo=bar", "EnvVariable" ]` crawl through it and replace any argument without `=` with `"EnvVariable=EnvValue"` **if** a  value was returned by `resolver` function  (`os.Getenv` in prod code, mocked function in tests). If zero length value returned, it will be replaced with `"EnvVariable="`

Reply:
Thanks for explaining, cheers